examples/tsconfig.json

```diff
@@ -7,13 +7,16 @@
         "inlineSourceMap": true,
         "importHelpers":true,
         "experimentalDecorators": true,
-        "noEmit": true,
+        "noEmit": false,
         "noImplicitAny": true,
         "noImplicitThis": true,
         "strict": true,
         "declaration": false,
         "skipLibCheck": true,
         "strictNullChecks": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "allowUnreachableCode": false,
```

Comment:
These don't seem to be working for me. (But only in this project; in the packages themselves they work just fine.)